icgcget/commands/versions.py

```diff
@@ -0,0 +1,16 @@
+import logging
+from icgcget.clients.ega.ega_client import EgaDownloadClient
+from icgcget.clients.gdc.gdc_client import GdcDownloadClient
+from icgcget.clients.gnos.gnos_client import GnosDownloadClient
+from icgcget.clients.icgc.storage_client import StorageClient
+from icgcget.clients.pdc.pdc_client import PdcDownloadClient
+
+
+def versions_command(cghub_path, ega_access, ega_path, gdc_path, icgc_path, pdc_path, version_num):
+    logger = logging.getLogger()
+    PdcDownloadClient().version_check(pdc_path)
```

Comment:
I'm thinking the command should be responsible for formatting the output here maybe? Definitely `version_check` is the wrong name. If anything it should be called `show_version`, `print_version`, etc. Can you paste an example of the output?